cyhy/mailer/HttpsMessage.py

```diff
@@ -137,7 +142,11 @@ def __init__(
 
         """
         # This is the data mustache will use to render the templates
-        mustache_data = {"acronym": entity_acronym, "report_date": report_date}
+        mustache_data = {
```

Comment:
Again.

Reply:
This one actually is already sorted alphabetically. 😄